### PR TITLE
Blog App - Creating a Data Model

### PR DESCRIPTION
**Pull request by raminka13**
Ruby Rails
Reference [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).
- Created a new app. `blog-app`.
- Initialized the project with Git.
- Postgres database s created in Rails.
- Built schema from migrations.
    - Created and ran necessary migration files.
    - Table and column names should match the ERD diagram.
        - note: photo for users table should be a link to an image
    - Foreign keys should be included.
    - All columns that are foreign keys should have a corresponding index.

### CODE REVIEW

File: db/migrate/20221018155708_create_post.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.bigint :author_id
```
Comment (mikethreels) [inline db/migrate/20221018155708_create_post.rb]: - there should be a reference to the author instead.

File: db/migrate/20221018155834_create_comments.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+
+      t.timestamps
+    end
```
Comment (mikethreels) [inline db/migrate/20221018155834_create_comments.rb:7]: - [ ] there should be a reference to author and post.

File: db/migrate/20221018191704_add_author_id_to_likes.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class AddAuthorIdToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_column :likes, :author_id, :bigint
+    add_index :likes, :author_id
+  end
```
Comment (mikethreels) [inline db/migrate/20221018191704_add_author_id_to_likes.rb]: - [ ] there should be a reference to author and post.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Raminka, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20221018155708_create_post.rb (hunk 1 above)
Comment (raminka13) [reply db/migrate/20221018155708_create_post.rb]: Hey @mikethreels did you check the `Schema.rb` ?

File: db/migrate/20221018155834_create_comments.rb (hunk 2 above)
Comment (raminka13) [reply db/migrate/20221018155834_create_comments.rb:7]: Hey @mikethreels did you check the `Schema.rb` ?

File: db/migrate/20221018191704_add_author_id_to_likes.rb (hunk 3 above)
Comment (raminka13) [reply db/migrate/20221018191704_add_author_id_to_likes.rb]: Hey @mikethreels did you check the `Schema.rb` ?

Comment (mikethreels) [reply db/migrate/20221018191704_add_author_id_to_likes.rb]: @raminka13, my apologies i think i made a mistake, please request another review

Comment (raminka13) [reply db/migrate/20221018191704_add_author_id_to_likes.rb]: It's Ok. I refactored the references migrations to be simpler, based on the Schema.rb it works.
Thanks.

**Review by mikethreels — APPROVED** [review]
Hi Raminka, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- good communication
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**